src/responder/component.ts

```diff
@@ -115,7 +115,8 @@ export class ResponderAppointmentReducer implements StateReducer<ResponderAppoin
 export enum ResponderActionKind {
     ReEnqueueMissingItems = 1,
     TxMined = 2,
-    EndResponse = 3
+    EndResponse = 3,
+    ResponderLowBalance = 4
```

Comment:
I think this should be renamed to `CheckResponderBalance`, since when the action is issued we dont know whether the balance is low or not.